pyproject.toml

```diff
@@ -36,7 +36,8 @@ output = "reports/coverage/coverage.xml"
 
 [tool.black]
 line-length = 88
-target-version = ['py37', 'py38']
+skip_magic_trailing_comma = true
```

Comment:
If you have this:

```
[very, long, list]
```
black will format it to:


```
[
    very,
    long,
    list,
]
```

If you then remove items but forget to remove the trailing comma, then you'll still get this expanded list. With that param set to true, then:

```
[
    very,
]
```
Will be formatted to:
```
[very]
```

Instead of staying expanded.